project-base/storefront/components/Layout/Header/Navigation/Navigation.tsx

```diff
@@ -7,7 +7,7 @@ export type NavigationProps = {
 
 export const Navigation: FC<NavigationProps> = ({ navigation }) => {
     return (
-        <ul className="relative hidden w-full lg:flex lg:gap-6 xl:gap-12">
+        <ul className="relative hidden w-full lg:flex">
```

Comment:
Not sure if this is what we want. Looking at the new design in Figma, I can see that the gap between menu items is set to 40px (tailwind size 10).